content/en/docs/components/katib/overview.md

```diff
@@ -87,83 +86,82 @@ part of the form for submitting a NAS job from the Katib UI:
 
 ## The Katib project
 
-Katib is a Kubernetes-based system for hyperparameter tuning and neural 
-architecture search. Katib supports a number of ML frameworks, including 
+Katib is a Kubernetes-based system for hyperparameter tuning and neural
+architecture search. Katib supports a number of ML frameworks, including
 TensorFlow, MXNet, PyTorch, XGBoost, and others.
 
-The [Katib project](https://github.com/kubeflow/katib) is open source. 
+The [Katib project](https://github.com/kubeflow/katib) is open source.
 The [developer guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md)
 is a good starting point for developers who want to contribute to the project.
```

Comment:
I think lines 87-95 about "the Katib project" should be right in the beginning of `overview.md` (see earlier comment).